packages/docs/src/routes/Anatomy.tsx

```diff
@@ -1,7 +1,11 @@
 import { Link } from 'react-router-dom'
 import PageHeader from '../components/PageHeader'
 // import { Grid, Column, OrderedList, ListItem } from '@carbon/react'
-// import './Anatomy.scss'
+// import styled from 'styled-components'
+
+// const DiagramImage = styled.img`
+// 	width: 100%;
+// `
```

Comment:
do we need this?

Reply:
Same as my last comment. The styles are there to support the content that was commented out. If we ever bring that content back, this CSS will be needed.